src/DI/CzechPostExtension.php

```diff
@@ -27,10 +29,26 @@ class CzechPostExtension extends CompilerExtension
 		],
 	];
 
+	public function getConfigSchema(): Nette\Schema\Schema
+	{
+		return Expect::structure(
+			[
+				'http' => Expect::structure([
+					'base_uri' => Expect::string('https://online.postservis.cz/'),
+					'auth' => Expect::array(['', '']),
```

Comment:
Why `auth` key is array with empty values?

Reply:
Don't know ... just used values from old `$defaults` property.

Reply:
I think it's the format expected by guzzle. I would go for explicit `user` and `password` keys and map it to the `[$config['http']['auth']['user'], $config['http']['auth']['password']]`

```phpp
$res = $client->request('GET', 'https://example.com', [
    'auth' => ['user', 'pass']
]);
```